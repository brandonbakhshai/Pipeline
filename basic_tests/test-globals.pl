global float x = 1.1;

global int a = 10;

function void test(){
    print_float(x);
    print_int(a);
}

test();
