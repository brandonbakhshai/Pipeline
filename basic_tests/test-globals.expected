1.100000
10
